code/game/dna/mutations/disabilities.dm

```diff
@@ -0,0 +1,549 @@
+	//List of swappable words. Normal word first, chav word second.
+	var/static/list/chavlinks = list(
+	"arrest" = "nick",
+	"arrested" = "nicked",
+	"ass" = "arse",
+	"bad" = "pants",
+	"bar" = "spoons",
+	"brain" = "noggin",
+	"break" = "smash",
+	"broke" = "smashed",
+	"broken" = "gone kaput",
+	"comdom" = "knob'ed",
+	"cool" = "ace",
+	"crazy" = "well mad",
+	"cup of tea" = "cuppa",
+	"destroyed" = "rekt",
+	"dick" = "prat",
+	"disappointed" = "gutted",
+	"disgusting" = "minging",
+	"disposals" = "bins",
+	"drink" = "bevvy",
+	"engineer" = "sparky",
+	"excited" = "jacked",
+	"fight" = "scuffle",
+	"food" = "nosh",
+	"friend" = "blud",
+	"fuck" = "fook",
+	"get" = "giz",
+	"girl" = "bird",
+	"go away" = "jog on",
+	"good" = "mint",
+	"great" = "bangin'",
+	"happy" = "chuffed",
+	"hello" = "orite",
+	"hi" = "sup",
+	"idiot" = "twit",
+	"isn't it" = "innit",
+	"kill" = "bang",
+	"killed" = "banged",
+	"man" = "bloke",
+	"mess" = "shambles",
+	"mistake" = "cock-up",
+	"murder" = "hench",
+	"murdered" = "henched",
+	"no" = "naw",
+	"really" = "propa",
+	"robust" = "'ard",
+	"run" = "leg it",
+	"sec" = "cops",
+	"security" = "coppers",
+	"silly" = "daft",
+	"steal" = "nick",
+	"stole" = "nicked",
+	"surprised" = "gobsmacked",
+	"suspicious" = "dodgy",
+	"tired" = "knackered",
+	"wet" = "moist",
+	"what" = "wot",
+	"window" = "windy",
+	"windows" = "windies",
+	"yes" = "ye",
+	"yikes" = "blimey",
+	"your" = "yur"
+	)
+
+/datum/mutation/disability/speech/chav/New()
+	..()
+	block = GLOB.chavblock
+
+/datum/mutation/disability/speech/chav/on_say(mob/M, message)
+	var/static/regex/R = regex("\\b([chavlinks.Join("|")])\\b", "g")
+	message = R.Replace(message, /datum/mutation/disability/speech/chav/proc/replace_speech)
+	return message
+
+/datum/mutation/disability/speech/chav/proc/replace_speech(matched)
+	return chavlinks[matched]
+
+// WAS: /datum/bioEffect/swedish
+/datum/mutation/disability/speech/swedish
+	name = "Swedish"
+	desc = "Forces the language center of the subject's brain to construct sentences in a vaguely norse manner."
+	activation_messages = list("You feel Swedish, however that works.")
+	deactivation_messages = list("The feeling of Swedishness passes.")
+
+/datum/mutation/disability/speech/swedish/New()
+	..()
+	block = GLOB.swedeblock
+
+/datum/mutation/disability/speech/swedish/on_say(mob/M, message)
+	// svedish
+	message = replacetextEx(message,"W","V")
+	message = replacetextEx(message,"w","v")
+	message = replacetextEx(message,"J","Y")
+	message = replacetextEx(message,"j","y")
+	message = replacetextEx(message,"A",pick("Å","Ä","Æ","A"))
+	message = replacetextEx(message,"a",pick("å","ä","æ","a"))
+	message = replacetextEx(message,"BO","BJO")
+	message = replacetextEx(message,"Bo","Bjo")
+	message = replacetextEx(message,"bo","bjo")
+	message = replacetextEx(message,"O",pick("Ö","Ø","O"))
+	message = replacetextEx(message,"o",pick("ö","ø","o"))
+	if(prob(30) && !M.is_muzzled())
```

Comment:
Surely `is_muzzled()` is checked before or after this right?

Reply:
Not a clue; this was already this way; I didn't alter it.